### PR TITLE
Update for forecast

### PR DESCRIPTION
**Pull request by Tommatheussen**
**Description:**
Added forecast possibility, related to the `weather` platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4721

### CODE REVIEW

File: source/_components/weather.openweathermap.markdown [hunk 1]
```diff
@@ -25,13 +25,15 @@ weather:
...
     longitude: 7.1234
+    forecast: True
 ```
 
 Configuration variables:
 
 - **api_key** (*Required*): Your API key for http://openweathermap.org/.
 - **latitude** (*Optional*): Latitude of the location to display the weather. Defaults to the latitude in your your `configuration.yaml` file. 
 - **longitude** (*Optional*): Longitude of the location to display the weather. Defaults to the longitude in your `configuration.yaml` file.
+- **forecast** (*Optional*): Wether the plugin should retrieve forecast data. Defaults to False.
```
Comment (fabaff) [inline source/_components/weather.openweathermap.markdown]: The correct term would be "platform" but I would like to suggest to leave it out.

Comment (Tommatheussen) [inline source/_components/weather.openweathermap.markdown]: This depends a bit wether we're going to keep this option, or just turn it always on.
I'll update accordingly to the decision on this.

**Comment by Tommatheussen** [comment]
Closing this as the current behaviour in the PR (https://github.com/home-assistant/home-assistant/pull/4721) has changed to always include the forecast data

File: source/_components/weather.openweathermap.markdown (hunk 1 above)
Comment (rpitera) [inline source/_components/weather.openweathermap.markdown]: Hate to be 'that' guy, but 'whether' is misspelled in the forecast option. A 'wether' is actually a castrated ram. LOL

Comment (Tommatheussen) [inline source/_components/weather.openweathermap.markdown]: Hehe, thanks for pointing this out, another improvement to my English :D

Comment (rpitera) [inline source/_components/weather.openweathermap.markdown]: Thanks for doing this PR - I'm looking for a good forecast component!

**Pull request closed without merging**